velox/exec/AggregateCompanionAdapter.cpp

```diff
@@ -475,6 +487,7 @@ bool CompanionFunctionsRegistrar::registerExtractFunctionWithSuffix(
         std::move(factory),
         exec::VectorFunctionMetadataBuilder()
             .defaultNullBehavior(false)
+            .isCompanionFunction(metadata.isCompanionFunction)
```

Comment:
Do we really need all these API changes to pass through the metadata with the hard coded companion flag set?
The `VectorFunctionMetadataBuilder` is called here from the `CompanionFunctionsRegistrar`. So by the very nature would the `CompanionFunctionsRegistrar` register functions that are not companion functions?
As a result wouldn't we be able to set the flag when the `VectorFunctionMetadataBuilder` is built here as `true`?

Reply:
Reverted the change to pass metadata here, thanks!